### DISCUSSION TITLE
503  Maintenance Mode  This website is currently undergoing maintenance. Please try again later.

### DISCUSSION THREAD

**Discussion by TarJae** [rubenarslan/formr.org#531] [General]
Using <https://workshops.formr.org/admin/survey/add_survey>
I can no longer access my account. Currently, I'm getting a 503 error.

Maintenance Mode
This website is currently undergoing maintenance. Please try again later.

May I kindly inquire how long the maintenance is expected to last before I can resume work on my project?

**Comment by JonathanSkorpil** [comment]
I'm having the same issue... I'm currently writing my bachelor thesis on a study I build in formR Workshop and I can't excess my run or data...

**Comment by cyriltata** [comment]
Dear all

workshops.formr.org should be used only for workshops and testing  purposes
and not to conduct real studies. We recently experienced a serious data
loss incident on that server and this will not be a problem if there only
test data there. If you were running a serious study please kindly ping me
privately with the name of your surveys and I will try my best to recover
the data for you.

Best regards
Cyril

**Reply by JonathanSkorpil** [reply]
I'm sorry, I'm quite new on GitHub and all that, how do I ping you?

**Reply by cyriltata** [reply]
@JonathanSkorpil you can send me an email.

**Reply by JonathanSkorpil** [reply]
I did, maybe I found the wrong mailing Adress, is "ctata@uni-goettingen.de" not correct?

**Comment by blumworx** [comment]
Well, it's definitely impossible to run FormR. Any updates? Thanks a lot and let's hope everything turns out as it was!

**Reply by rubenarslan** [reply]
I don't know when/if @cyriltata plans to put workshops.formr.org back up. But he never intended this to be used for real studies, only for workshops to train people to use formr. If you want to use formr, we recommend you host it yourself, with our assistance.

**Reply by blumworx** [reply]
Of course, Ruben! Thanks for your reply.
